gravitee-apim-console-webui/src/management/api/documentation-v4/components/documentation-edit-page/documentation-edit-page.component.html

```diff
@@ -103,6 +103,15 @@
                       </div>
                     </gio-form-label>
                   </gio-form-slide-toggle>
+                  <gio-form-slide-toggle>
+                    <mat-slide-toggle gioFormSlideToggle data-test-id="disable-syntax-highlight" formControlName="disableSyntaxHighlight" />
+                    <gio-form-label>
+                      <div>Disable Styling for Response body for large payload json data</div>
+                      <div class="mat-body-2">
+                        Renders API response without swagger-ui styling for large payload for better performance.
+                      </div>
```

Comment:
```suggestion
                      <div>Disable response body styling for large JSON payloads</div>
                      <div class="mat-body-2">
                        Displays API responses without Swagger UI styling to optimize performance when handling large payloads.
                      </div>
```
If this works for you, then the changes can also be applied for v2 API settings 👍

Reply:
agreed. i did the changes for both.